apps/percona/functional/backup_and_restore/test.yml

```diff
@@ -35,6 +35,21 @@
             dbpassword: 'k8sDem0'
             dbname: "backup"
 
+        - block:
+            
+           - name: Getting the application deployment name
+             shell: kubectl get deploy -n {{ app_ns }} -l {{ app_label }} -o jsonpath='{.items[0].metadata.name}'
+             register: app_deploy_name
+
+           - name: Fetching the volume name of application
+             shell: kubectl get deploy {{ app_deploy_name.stdout }} -n {{ app_ns }} -o jsonpath='{.spec.template.spec.volumes[0].name}'
```

Comment:
Please add a comment here that annotation is limited to one volume only. Mentioning as `TODO` to remove this limit, will be better.